plugin.rb

```diff
@@ -872,6 +872,15 @@ module ::DiscourseAssign
     end
   end
 
+  on(:group_destroyed) do |group, user_ids|
+    user_ids.each do |user_id|
+      user = User.find(user_id)
```

Comment:
```suggestion
      user = User.find_by(id: user_id)
```
Later you have `if user` so we don't want error when a user does not exist